src/handlers/motions/motionCreated/handlers/networkUpgrade.ts

```diff
@@ -8,11 +9,13 @@ import { createMotionInDB } from '../helpers';
 export const handleNetworkUpgradeMotion = async (
   event: ContractEvent,
   parsedAction: TransactionDescription,
+  gasEstimate: BigNumber,
 ): Promise<void> => {
   const { name, args: actionArgs } = parsedAction;
   const newVersion = actionArgs[0];
   await createMotionInDB(event, {
     type: motionNameMapping[name],
     newColonyVersion: toNumber(newVersion),
+    gasEstimate: gasEstimate.toString(),
```

Comment:
I have changed it so we estimate gas for all motion created actions in the block ingestor. This allows us to increase our estimates for actions we know consume more gas, like multicall